test/mlir/unittests/ExecutionEngine/GPU/GpuOclRuntimeTest.cpp

```diff
@@ -220,7 +220,7 @@ TEST(GpuOclRuntime, TestAddDynamic) {
 }
 
 TEST(GpuOclRuntime, TestMatmulAddStatic) {
-  struct Test : TestMatmulAdd<64, 128> {
+  struct Test : TestMatmulAdd<128, 256> {
```

Comment:
the new answer for this test 21512 (should be corrected [here](https://github.com/intel/graph-compiler/blob/ddc3dd3a51782384a0916d6e4c2d1ee88951ba68/test/mlir/unittests/ExecutionEngine/GPU/GpuOclRuntimeTest.cpp#L170))